### PR TITLE
feat(Injector): disable hidden api policy

### PR DESCRIPTION
**Pull request by DiamondMiner88**
Very cursed code, but it works
Any further reflection calls on hidden methods will succeed during the apps lifetime

### CODE REVIEW

**Comment by Vendicated** [comment]
add comments explaining what this does

File: Injector/src/main/java/com/aliucord/injector/Injector.java [hunk 1]
```diff
@@ -208,4 +211,35 @@ private static boolean pruneArtProfile(Context ctx) {
...
+     * This works because the VM thinks its internals calling the hidden method,
+     * since you're invoking reflection methods using reflection.
+     */
+    private static void disableHiddenApiPolicy() {
+        try {
+            var mForName = Class.class.getDeclaredMethod("forName", String.class);
+            var mGetDeclaredMethod = Class.class.getDeclaredMethod("getDeclaredMethod", String.class, Class[].class);
+
+            var cVMRuntime = mForName.invoke(null, "dalvik.system.VMRuntime");
+            var mGetRuntime = (Method) mGetDeclaredMethod.invoke(cVMRuntime, "getRuntime", null);
```
Comment (Vendicated) [inline Injector/src/main/java/com/aliucord/injector/Injector.java:228]: shouldn't this be 
```suggestion
            var mGetRuntime = (Method) mGetDeclaredMethod.invoke(cVMRuntime, "getRuntime", new Class[0]);
```

**Comment by Vendicated** [comment]
have u tested that this indeed does bypass hidden api?

**Comment by DiamondMiner88** [comment]
I wouldn't have pr'd it if it didnt

**Pull request merged**